spring-graphql-api/src/main/resources/application.properties

```diff
@@ -1,11 +1,10 @@
-
 spring.graphql.graphiql.enabled=true
 spring.graphql.schema.introspection.enabled=true
 org.neo4j.migrations.transaction-mode=PER_STATEMENT
 spring.neo4j.uri=bolt://localhost:7687
 spring.neo4j.authentication.username=neo4j
-spring.graphql.cors.allowed-origins=http://localhost:3000
 
-okta.oauth2.issuer=
-okta.oauth2.client-id=
-okta.oauth2.audience=
\ No newline at end of file
+spring.graphql.cors.allowed-origins=http://localhost:3000
+okta.oauth2.issuer=https://dev-06bzs1cu.us.auth0.com/
+okta.oauth2.client-id=HBOg3YPUl1QSaLq6yoMzUaN3Qct75Tjq
```

Comment:
Yes, maybe something like this:

```properties
okta.oauth2.issuer=https://<your-oauth-domain>/
okta.oauth2.client-id=<client-id>
```

I think it's also fine to leave it as-is because it might provide a better example. I've already deleted the app from my Auth0 instance.